src/services/logger/logger.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+import Pino from 'pino'
+const logger = Pino()
+
+logger.info(`Hi! Here is my first massage`)
+
+logger.setLevel = function setLevel (newLevel: any) {
+    logger.level = newLevel;
+}
+logger.on('level-change', () => {
+  logger.info(`the Level defined as ${logger.level}`)
```

Comment:
```suggestion
  logger.debug(`Logger: Level changed to ${logger.level}`)
```

Reply:
If you not define any env-level, the event will still be activated on 'info'.
That's why I wrote ' defined as'